common/djangoapps/xblock_django/models.py

```diff
@@ -42,16 +42,6 @@ def is_block_type_disabled(cls, block_type):
         return block_type in config.disabled_blocks.split()
 
     @classmethod
-    def disabled_block_types(cls):
```

Comment:
Note to reviewers: this model add associated unit tests will be deleted in a future story (after deprecation is moved over to XBlockConfiguration).